bin/cli-flags.js

```diff
@@ -85,15 +85,15 @@ module.exports = {
       describe: 'HTTP/2, must be used with HTTPS',
     },
     {
-      name: 'content-base',
+      name: 'static-directory',
       type: String,
       describe: 'A directory or URL to serve HTML content from.',
       group: RESPONSE_GROUP,
     },
     {
-      name: 'watch-content-base',
+      name: 'static-watch',
       type: Boolean,
-      describe: 'Enable live-reloading of the content-base.',
+      describe: 'Enable live-reloading of the static directory.',
```

Comment:
I think we should not provide here a lot of stuff, `--static=path/to/directory` and `--no-static` is enough, for other complex examples you need to use the configuration

Reply:
So we will get rid of the `serve-index` CLI flag?

Reply:
Also, it seems our CLI system does not work well with a `no-static` flag, since `static` would have to be a string flag, not sure how to work around it without a hacky method